plasmapy/physics/transport/collisions.py

```diff
@@ -14,6 +14,8 @@
 
 from plasmapy.constants import (c, m_e, k_B, e, eps0, pi, hbar)
 from plasmapy.atomic import (particle_mass, integer_charge)
+from plasmapy import atomic
+from plasmapy.physics import parameters
```

Comment:
The line below this already imports from parameters - seems redundant.

Reply:
I'll get back to you on this - it's a namespace visibility and Sphinx automodapi thing that I think should probably be written down and visible. Long story short, the imports we currently do are fine if we remember a few tricks.

Reply:
Perhaps we should change to `import blah as bla` format throughout? I think that's the recommend method, though I don't know how that meshes with Sphinx.